main.cpp

```diff
@@ -134,20 +144,31 @@ struct Bank
     float moneyWithdrawn = 100.25f;
     float interestPaid = 03.03f;
     float debtOwed = 2000.05f;
-    float Paid = 1500.25f;
+    float paid = 1500.25f;
+    Bank ();
     float holdMoney(float amount);
     void processTransactions();
     float produceCashBalance(float balance);
 };
 
+Bank::Bank()
+{
+    moneyDeposited = 0;
+    moneyWithdrawn = 0;
+    interestPaid = 0;
+    debtOwed = 0;
+    paid = 0;
```

Comment:
This overwrites the default values that your variables have. 
remove these value assignments.
- [ ] revise